components/ca-property.js

```diff
@@ -0,0 +1,338 @@
+define([
+    './helpers/component-support.js',
+    './helpers/create-element-legacy.js',
+    './helpers/escape-html.js',
+    './helpers/clear-element.js',
+    './helpers/resolve-enum-titles.js',
+    './helpers/parse-iso-local-date.js',
+    'document-register-element',
+    'intl-api'
+], (componentSupport, createElement, escapeHtml, clearElement, resolveEnumTitles, parseIsoLocalDate) => {
+    /**
+     * Property class for ca-property web component.
+     * @exports ca-property
+     * @description A custom HTML element (Web Component) that can be created using
+     * document.createElement('ca-property') or included in a HTML page as an element.
+     * @extends HTMLElement
+     */
+    class Property extends HTMLElement {
+        /**
+         * Executes when the element is attached to the DOM
+         * @returns {void} void
+         */
+        attachedCallback() {
+            this.render();
+        }
+
+        /**
+         * observedAttributes - get a list of attributes that should effect re-render.
+         * @return {Array<string>} list of attributes that effect a re-render.
+         */
+        static get observedAttributes() {
```

Comment:
What is the purpose of observedAttributes returning 'title' when you are specifically testing for 'title' in attributeChangedCallback?